npm-react/addons.js

```diff
@@ -1,8 +1,8 @@
 var warning = require('./lib/warning');
 warning(
   false,
-  'require("react/addons") is deprecated. ' +
-  'Access using require("react/addons/{addon}") instead.'
+  'require(\'react/addons\') is deprecated. ' +
```

Comment:
That isn't actually. My bad regexp codemod caught it, and I figured it wouldn't hurt to change it. Let me know if you want me to revert it.